src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/config/OpenApiConfiguration.kt

```diff
@@ -0,0 +1,31 @@
+package uk.gov.justice.digital.hmpps.nomisuserrolesapi.config
+
+import io.swagger.v3.oas.models.OpenAPI
+import io.swagger.v3.oas.models.info.Contact
+import io.swagger.v3.oas.models.info.Info
+import io.swagger.v3.oas.models.servers.Server
+import org.springframework.boot.info.BuildProperties
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class OpenApiConfiguration(buildProperties: BuildProperties) {
+  private val version: String = buildProperties.version
+
+  @Bean
+  fun customOpenAPI(): OpenAPI = OpenAPI()
+    .servers(
+      listOf(
+        Server().url("https://nomis-user-roles-api.prison.service.justice.gov.uk/").description("Prod"),
+        Server().url("https://nomis-user-roles-api-preprod.prison.service.justice.gov.uk").description("PreProd"),
+        Server().url("https://nomis-user-roles-api-dev.prison.service.justice.gov.uk/").description("Development"),
+        Server().url("http://localhost:8080").description("Local"),
```

Comment:
does this help adding localhost? Can you actually use the Try It button?

Reply:
Good point